### PR TITLE
[RHPAM-3984] Fix prod standalone jar due to migration EAP bootable jar

### PR DESCRIPTION
**Pull request by AntStephenson**
**JIRA**: https://issues.redhat.com/browse/RHPAM-3984

This PR updates the productization of the standalone jar to execute the wildfly jar plugin in hollow jar mode referencing the produced productized war. Also included is a property generation plugin execution as maven properties are not exposed by default in CLI sessions so this file is then passed in to be made available.

**referenced Pull Requests**:

Will raise PR for main once this is approved

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: business-central-parent/business-central-distribution-wars/business-central/scripts/set-rhdm-props.cli [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/system-property=org.kie.workbench.profile:add(value=FORCE_PLANNER_AND_RULES)
+/system-property=org.kie.workbench.profile:read-resource
```
Comment (almope) [inline business-central-parent/business-central-distribution-wars/business-central/scripts/set-rhdm-props.cli:2]: Reading resources is not needed here. Please remove it.

**Review by almope — APPROVED** [review]
I've added a comment for removing an unnecessary thing. The rest looks good to me.

File: business-central-parent/business-central-distribution-wars/business-central/scripts/set-rhdm-props.cli (hunk 1 above)
Comment (AntStephenson) [reply business-central-parent/business-central-distribution-wars/business-central/scripts/set-rhdm-props.cli:2]: Yeah you're right, I put them in for debugging.. should remove, thanks for looking ;-)

**Review by Rikkola — APPROVED** [review]

**Comment by AntStephenson** [comment]
Jenkins retest this

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me. Thanks 🙂

**Comment by AntStephenson** [comment]
jenkins retest this

**Comment by mareknovotny** [comment]
jenkins retest this

**Comment by mareknovotny** [comment]
have you tried to build it outside this CI as it doesn't use productized profile. We can't rely on CI here

**Comment by mareknovotny** [comment]
we need also backport this for rhpam only in main branch @AntStephenson

**Comment by AntStephenson** [comment]
@mareknovotny Yep, I wrote in the original comment I will raise it after this is merged. In fact I will raise one after CR1 is built. ;-)

**Pull request merged**